galata/test/documentation/utils.ts

```diff
@@ -56,3 +58,19 @@ export function positionMouse(position: { x: number; y: number }): string {
       stroke="#ffffff" stroke-width=".54745" style="paint-order:markers fill stroke" />
 </svg>`;
 }
+
+export async function setLeftSidebarWidth(
+  page: Page,
+  width = 251
+): Promise<void> {
+  const splitHandle = await page.$('.lm-SplitPanel-handle');
+  const handleBBox = await splitHandle.boundingBox();
+
+  await page.mouse.move(
+    handleBBox.x + 0.5 * handleBBox.x,
+    handleBBox.y + 0.5 * handleBBox.height
+  );
+  await page.mouse.down();
+  await page.mouse.move(33 + width, handleBBox.y + 0.5 * handleBBox.height);
```

Comment:
Do we need to move along the `y` axis?

Wondering if this might be causing the selection mentioned in https://github.com/jupyterlab/jupyterlab/pull/12123#issuecomment-1054018426.

Reply:
The position is the absolute one. So there is no y motion. But the selection makes me wonder if we really are catching the handle.

Reply:
Ah right. Then yes it's possible the handle is indeed not caught. Posting the Playwright captured video here for reference:

https://user-images.githubusercontent.com/591645/156006004-4e557e45-480c-4969-8ea2-fb1c42890b8f.mp4